src/center.css

```diff
@@ -0,0 +1,30 @@
+
+.center{
+    display: flex;
+    transform: translate(0,+50%);
+
+  }
+
+.resized{
+    width: 400px;
+    height: 200px;
+}
+
+.top{
+  float:top;
```

Comment:
Float is not widely used anymore. Think about changing it to [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).